src/components/emailerFilterControls/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import moment from 'moment';
+import { EmailerFilters } from '../../containers/email/types';
+import {
+  Collapse,
+  Slider,
+  DatePicker,
+  AutoComplete,
+  Button,
+  message,
+} from 'antd';
+import { SliderMarks } from 'antd/lib/slider';
+import styled from 'styled-components';
+import { Neighborhoods } from '../../assets/content';
+import apiClient from '../../api/apiClient';
+import { CloseOutlined } from '@ant-design/icons';
+
+const StyledCollapse = styled(Collapse)`
+  max-width: 400px;
+`;
+
+const StyledRangePicker = styled(DatePicker.RangePicker)`
+  margin-right: 15px;
+`;
+
+const autoCompleteStyles = { width: 200, marginRight: 15 };
+
+interface EmailerFilterControlsProps {
+  filters: EmailerFilters;
+  setFilters: React.Dispatch<React.SetStateAction<EmailerFilters>>;
+}
+
+const MAX_COUNT = 10;
+
+function activityCountRange(filters: EmailerFilters): [number, number] {
+  return [filters.activityCountMin, filters.activityCountMax || MAX_COUNT + 1];
+}
+
+function formatDates(
+  start?: string,
+  end?: string,
+): [moment.Moment | null, moment.Moment | null] | undefined {
+  if (start && end) return [moment(start), moment(end)];
+}
+
+function disabledDate(current: moment.Moment): boolean {
+  // Can not select future days
+  return current > moment().endOf('day');
+}
+
+const activityCountLabels: SliderMarks = {
+  0: '0',
+  [MAX_COUNT + 1]: `${MAX_COUNT}+`,
+};
+
+const neighborhoodOptions = Object.values(Neighborhoods)
+  .map((value) => {
+    return { value };
+  })
+  .sort();
+
+const EmailerFilterControls: React.FC<EmailerFilterControlsProps> = ({
+  filters,
+  setFilters,
+}) => {
+  const [neighborhoodSearch, setNeighborhoodSearch] = useState<string>('');
+  const [namesSearch, setNamesSearch] = useState<string>('');
+
+  const [commonNameOptions, setCommonNameOptions] = useState<
+    { value: string }[]
+  >([]);
+
+  useEffect(() => {
+    async function getNames() {
+      const names = await apiClient.getAllCommonNames();
+      setCommonNameOptions(
+        names.map((name) => {
+          return { value: name };
+        }),
+      );
+    }
+    getNames();
+  }, []);
+
+  const onAddFilter = useCallback(
+    (
+      key: 'neighborhoods' | 'commonNames',
+      options: { value: string }[],
+      searchValue: string,
+      warning: string,
+    ) => {
+      return () => {
+        if (options.map((option) => option.value).includes(searchValue)) {
+          if (!filters[key].includes(searchValue)) {
+            setFilters({ ...filters, [key]: [...filters[key], searchValue] });
+            key === 'neighborhoods'
+              ? setNeighborhoodSearch('')
+              : setNamesSearch('');
+          }
+        } else {
+          message.warn(warning);
+        }
+      };
+    },
+    [filters, setFilters],
+  );
+
+  return (
+    <StyledCollapse ghost={true}>
```

Comment:
❓Just curious, what does the `ghost` prop do? Also, just an fyi that if the value of a prop/attribute is `true`, we don't actually need to specify that (in other words, `ghost={true}` can just be `ghost`)

Reply:
Ghost makes the collapse panels borderless and transparent, which I think matches the figma design a little better: 

![image](https://user-images.githubusercontent.com/68914997/232233238-c133ef0a-4b8d-4f67-b2b8-51e9a32f7d04.png)